odoo/fields.py

```diff
@@ -884,13 +884,9 @@ def update_db(self, model, columns):
         # create/update the column, not null constraint; the index will be
         # managed by registry.check_indexes()
         self.update_db_column(model, column)
-        skip_recomputation = self.update_db_notnull(model, column)
+        initialized = self.update_db_notnull(model, column)
```

Comment:
:+1: 
I felt that ``skip_recomputation`` is something like a variable with negation